### DISCUSSION TITLE
"error": "coapp_failure",

### DISCUSSION THREAD

**Discussion by david-rahrer** [aclap-dev/video-downloadhelper#476] [Help and Q&A] [answered]
### Your question

```
{
  "version": "9.0.1.9a1",
  "target": "mozilla",
  "channel": "beta",
  "page_url": "https://www.voicesofthesilenced.com/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 000001b7987468c0] Error when loading first segment 'https://videocore-issuesorg.cdn.ypt.me/videos/_YPTuniqid_5feb4c80233609.90163029/res1080/index0.ts'\r\n[in#0 @ 000001b798746740] Error opening input: Invalid data found when processing input\r\nError opening input file https://videocore-issuesorg.cdn.ypt.me/videos/_YPTuniqid_5feb4c80233609.90163029/res1080/index.m3u8?token=eyJzYWx0IjoiaHR0cHM6XC9cL3ZpZGVvLmNvcmUtaXNzdWVzLm9yZ1wvIiwidG9rZW5UaW1lT3V0IjoiMjE2MDAiLCJmaWxlbmFtZSI6Il9ZUFR1bmlxaWRfNWZlYjRjODAyMzM2MDkuOTAxNjMwMjkiLCJ0aW1lIjoxNzA4OTUxODg0fQ==.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_9d305752-f86f-475b-918a-cd822f35a14c",
    "downloadable_id": "downloadable_6731491792097968"
  }
}
```
I tried this first on the current version before installing the beta - neither worked. I didn't see a beta version of the companion app. I'm open to checking on something else or testing. I'm on Firefox 125.0.3 64bit on Windows 11 if that helps. Please ignore the video topic, I'm a researcher and I need to download the video so I can cut clips for commentary or critique. I don't share those views. Thanks!

### Addon & CoApp version and details

version: 9.0.1.9a1
target: mozilla
channel: beta
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"cd94982bbfc949b9"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 9
}

**Comment by Mattmjs** [comment, marked as answer]
It's either another non standard streaming method or there is some protection put in place.
You mean the 1 hour 46 minute video correct ?

Well I was snooping around in the dev tools and somewhere in the response for this video there were notes saying:
_you cannot download videos we will not share the video source file_

I guess they did their job because it can't be downloaded it seems.

**Comment by david-rahrer** [comment]
Yeah, I've tried everything, even other extensions. In my experience, if VDH doesn't get it, nothing will. Thanks for your quick response. I'll monitor in case a solution pops up.
